index.bs

```diff
@@ -1188,6 +1188,24 @@ partial interface HTMLIFrameElement {
     Note: This algorithm should be called when a feature policy has
     been <a>violated</a>.
   </section>
+  <section>
+    <h3 id="should-request-be-allowed-to-use-feature">Should <var>request</var> be allowed to use <var>feature</var>?</h3>
+    <p>Given a feature (<var>feature</var>) and a  <a for="/">request</a> (<var>request</var>), this algorithm returns <code>true</code> if the request should be allowed to use <var>feature</var>, and <code>false</code> otherwise.</p>
+    <ol>
+      <li>Set |global object| to |request|’s <a for="request">client</a>’s <a for="environment settings object">global object</a>.</li>
```

Comment:
A request's client may be null, so this spec has to deal with that.

Reply:
In what contexts? If there’s no other way to get an origin in those contexts, I guess we'll return false.